Plugins/ExternalTexture/Readme.md

```diff
@@ -14,22 +14,26 @@ ExternalTexture is a plugin library that provided the functionality required to
 // Create graphics Device
 Babylon::Graphics::Configuration graphicsConfig {};
 graphicsConfig.Device = d3dDevice;
-Babylon::Graphics::Device babylonDevice = Babylon::Graphics::Device(config);
+Babylon::Graphics::Device babylonDevice{config};
 ```
 
 If the external texture will be used as a render target, there is no need to provide a width, height or Window values for the Configuration object.
 
-To set the ExternalTexture to be used as a render target by Babylon.js one must call a JS function that will assign the 
+To set the ExternalTexture to be used as a render target by Babylon.js one must call a JS function that will assign the native object to a JS texture.
 
 ```
-int width  // Your render target width.
-int height // Your render target height. 
+int width = 1024;  // Your render target width.
+int height = 768; // Your render target height. 
 
 std::promise<void> textureCreationSubmitted {};
 std::promise<void> textureCreationDone {};
 
+// Create an ExternalTexture from an ID3D12Resource.
+std::shared_ptr<Babylon::Plugins::ExternalTexture> externalTexture = std::make_shared<Babylon::Plugins::ExternalTexture>(d3d12Resource);
```

Comment:
Nit: Probably more readable to use `auto` here since type is pretty clear from the call to `make_shared`:
```suggestion
auto externalTexture = std::make_shared<Babylon::Plugins::ExternalTexture>(d3d12Resource);
```